controllers/ovndbcluster_controller.go

```diff
@@ -786,6 +806,69 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	return ctrl.Result{}, nil
 }
 
+// generateServiceConfigMaps - create create configmaps which hold service configuration
+func (r *OVNDBClusterReconciler) generateExternalConfigMaps(
+	ctx context.Context,
+	h *helper.Helper,
+	instance *ovnv1.OVNDBCluster,
+	serviceName string,
+	envVars *map[string]env.Setter,
+) error {
+	// Create/update configmaps from templates
+	cmLabels := labels.GetLabels(instance, labels.GetGroupLabel(serviceName), map[string]string{})
+	encapType := ""
+
+	externalEndpoint, err := instance.GetExternalEndpoint()
+	if err != nil {
+		return err
+	}
+
+	ovnController, err := ovnv1.GetOVNController(ctx, h, instance.Namespace)
+	if err != nil {
+		return err
+	} else if ovnController != nil {
+		encapType = ovnController.Spec.ExternalIDS.OvnEncapType
+	}
+
+	externalTemplateParameters := make(map[string]interface{})
+	externalTemplateParameters["OVNRemote"] = externalEndpoint
+	if encapType != "" {
+		externalTemplateParameters["OVNEncapType"] = encapType
```

Comment:
(Not action required, Observation) Long term, I think this config may be better separated into a separate ConfigMap, then both can be mounted / consumed by edpm-ansible role.